chia/server/address_manager_store.py

```diff
@@ -151,10 +198,8 @@ async def _deserialize(cls, peers_file_path: Path) -> AddressManager:
         peer_data: Optional[PeerDataSerialization] = None
         address_manager = AddressManager()
         start_time = timer()
-        try:
-            peer_data = await cls._read_peers(peers_file_path)
-        except Exception:
-            log.exception(f"Unable to deserialize peers from {peers_file_path}")
+        # if this fails, we can catch the exception above and try the other type of deserializing
```

Comment:
"above" means in the caller, yeah?

Reply:
Yes, I've amended this comment for clarity now